ui/DesignSystem/Primitive/Icon/Verified.svelte

```diff
@@ -1,8 +1,8 @@
 <script>
   import IconWrapper from "./IconWrapper.svelte";
 
-  export let dataCy = null;
-  export let style = null;
+  export let dataCy = "";
+  export let style = "";
```

Comment:
There is some work around required/optional props in sveltejs/svelte#5183, sveltejs/language-tools#273 and sveltejs/language-tools#319.